microsoft/testsuites/power/common.py

```diff
@@ -38,10 +37,10 @@ def is_distro_supported(node: Node) -> None:
             f"version {node.os.information.version}"
         )
 
-    if (
-        (isinstance(node.os, Redhat) and node.os.information.version < "8.3.0")
-        or (isinstance(node.os, Ubuntu) and node.os.information.version < "18.4.0")
-        or (isinstance(node.os, Suse) and node.os.information.version < "15.3.0")
+    if not (
+        (type(node.os) == Ubuntu and node.os.information.version >= "18.04.0")
```

Comment:
please fix this regression `(type(node.os) == Ubuntu and node.os.information.version >= "18.4.0")`

Reply:
fixed in #3533